### PR TITLE
Update ReleaseProcess.md

### PR DESCRIPTION
**Pull request by viorel15**
This adds info related to manual cut-off if necessary

### CODE REVIEW

**Comment by ilyapuchka** [comment]
https://github.com/babylonhealth/ios-playbook/pull/297

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 1]
```diff
@@ -30,7 +30,8 @@ There are usually two release engineers working at any given time. It goes witho
 ## 2. Release step-by-step
 
 ### Phase 1: Pre-release considerations
-1. To help expedite the release, any required feature flag changes should be done prior to the 2am cut-off on the Friday. If these are missed, a concession can be made so that the changes will be allowed to be made during the Friday. However, this should not be a regular occurrence as it requires intervention from the assigned release engineers to create new builds.
+1. To help expedite the release, any required feature flag changes should be done prior to the 2am cut-off on the Friday. If these are missed, a concession can be made so that the changes will be allowed to be made during the Friday. However, this should not be a regular occurrence as it requires your intervention to create new builds.
+If the automatic cut-off fails then this must be triggered manually: Run `/stevenson release_cutoff target:{appname}` in the build channel.
 
 ### Phase 2: Initiation
```
Comment (viorel15) [inline Cookbook/Technical-Documents/ReleaseProcess.md]: @ilyapuchka It seems that his step overlaps with the `release_cutoff` command. How does this change for other targets?

File: Cookbook/Technical-Documents/ReleaseProcess.md [hunk 2]
```diff
@@ -30,7 +30,8 @@ There are usually two release engineers working at any given time. It goes witho
 ## 2. Release step-by-step
 
 ### Phase 1: Pre-release considerations
-1. To help expedite the release, any required feature flag changes should be done prior to the 2am cut-off on the Friday. If these are missed, a concession can be made so that the changes will be allowed to be made during the Friday. However, this should not be a regular occurrence as it requires intervention from the assigned release engineers to create new builds.
+1. To help expedite the release, any required feature flag changes should be done prior to the 2am cut-off on the Friday. If these are missed, a concession can be made so that the changes will be allowed to be made during the Friday. However, this should not be a regular occurrence as it requires your intervention to create new builds.
+If the automatic cut-off fails then this must be triggered manually: Run `/stevenson release_cutoff target:{appname}` in the build channel.
```
Comment (sdaniel55) [inline Cookbook/Technical-Documents/ReleaseProcess.md]:
```suggestion
If the automatic cut-off fails then this must be triggered manually: Run `/stevenson release_cutoff target:{appname}` in the `#ios-build` channel.
```

**Review by danspndl — APPROVED** [review]
👌

**Comment by viorel15** [comment]
@ilyapuchka also created one

File: Cookbook/Technical-Documents/ReleaseProcess.md (hunk 1 above)
Comment (ilyapuchka) [reply Cookbook/Technical-Documents/ReleaseProcess.md]: we don't do automatic release cutoffs for any other target

**Pull request closed without merging**